src/content/blog/why-speak-at-jsheroes.md

```diff
@@ -0,0 +1,39 @@
+---
+author: alex-moldovan
+title: Why speak at JSHeroes
+published: Nov 8, 2024
+tags: ["cfp"]
+summary: A few reasons to convince you to apply as a speaker
+---
+
+In 2025 we’ll be running JSHeroes for the 7th time. In all our past editions we had amazing speakers on our stage. Maybe some of you reading this are already convinced and you want to be a speaker at our event, but for everyone else, let me try to articulate the "why".
+
+## Recognition, exposure and benefits
+
+Our conference has grown steadily and has a very good reputation among tech events. As we have a single track with generous breaks, you will be part of a rather small group of speakers that take the stage every year. Your talk will be seen live by 500+ attendees and then it will be available on our youtube channel to be shared with the entire world.
```

Comment:
```suggestion
Our conference has grown steadily and has a very good reputation among tech events. We have a single track with generous breaks. You will be part of a rather small group of speakers that take the stage every year. Your talk will be seen live by 500+ attendees and then it will be available on our YouTube channel for the entire world.
```